espnet/nets/chainer_backend/transformer/subsampling.py

```diff
@@ -15,32 +15,29 @@ class Conv2dSubsampling(chainer.Chain):
     def __init__(self, channels, idim, dims, dropout=0.1,
                  initialW=None, initial_bias=None):
         super(Conv2dSubsampling, self).__init__()
-        n = 1 * 3 * 3
-        stvd = 1. / np.sqrt(n)
-        layer = L.Convolution2D(1, channels, 3, stride=2, pad=1,
-                                initialW=initialW(scale=stvd),
-                                initial_bias=initial_bias(scale=stvd))
-        self.add_link('conv.0', layer)
-        n = channels * 3 * 3
-        stvd = 1. / np.sqrt(n)
-        layer = L.Convolution2D(channels, channels, 3, stride=2, pad=1,
-                                initialW=initialW(scale=stvd),
-                                initial_bias=initial_bias(scale=stvd))
-        self.add_link('conv.2', layer)
-        stvd = 1. / np.sqrt(dims)
-        layer = L.Linear(idim, dims, initialW=initialW(scale=stvd),
-                         initial_bias=initial_bias(scale=stvd))
-        self.add_link('out.0', layer)
         self.dropout = dropout
         with self.init_scope():
+            n = 1 * 3 * 3
```

Comment:
Can you add some comments about `n = 1 * 3 * 3`?

Reply:
ok